Gemfile

```diff
@@ -10,14 +10,19 @@ group :development, :test do
 end
 
 group :development do
-  gem 'binding_of_caller'
   gem 'better_errors'
+  gem 'binding_of_caller'
   gem 'bullet'
+  gem 'guard'
+  gem 'guard-bundler', require: false
+  gem 'guard-minitest', require: false
+  gem 'guard-rails', require: false
```

Comment:
I've found guard-rails kind of annoying 'cause I use pow and ended up removing it